examples/json/json.type.ts

```diff
@@ -24,11 +24,6 @@ export const Json = {
     source: `${__dirname}json.schema`,
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":6,"prLink":"https://app.graphite.dev/github/pr/skyleague/therefore/3"}} -->
<i class='graphite__hidden'>[Re: line 6]<br/></i>

This import is still generated<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/skyleague/therefore/3?utm_source=unchanged-line-comment">Graphite</a>.</span>